### DISCUSSION TITLE
call for co-moderators for the upcoming Krita Ai diffusion user group

### DISCUSSION THREAD

**Discussion by minsky91** [Acly/krita-ai-diffusion#906] [General]
Greetings everyone. With 53% of votes (8 out of 15), Discord has won as the hosting platform for our upcoming user group. I am glad to see so many people responding. Before launching the group there though, I would like to find someone with whom to share the workload and responsibilities that go with that. It’s never a good idea to be a sole moderator of a group, no matter how small. 

Anyone willing to be a co-moderator? If positive, send me a DM on Discord (I am minsky91 there), so we could further discuss how we split the duties, I am flexible on that. Also, what channels would be useful to create in the group, rules of conduct, and all the other questions involved. (To simplify things, we could use an existing group as a template.)

**Comment by lovemakerz** [comment]
Hello, 
I'm french, and i'm not very good in english language xD, i'm sorry.
i have allready 3 Discords in management, but i you need help, i'm here !
This project is awesome, 
We need a Discord for this !

**Reply by minsky91** [reply]
Great, thanks! I feel relieved to hear someone is willing to help, so we could actually get on implementing the idea. Your Discord experience will come really handy, since I myself am only recently there, relatively speaking. Would you send me a DM there, then? (my nic is ‘minsky91’) If you have an already existing Discord group in mind that we could use as a template for our own, that will accelerate the process.  

Any other suggestions, like which channels to create and ideas on the welcome message, are welcome! Whom do we want to invite? Where, on which social media apart from github and reddit’s /r/StableDiffusion should we place an ad about the creation of our group? (these are questions to **everyone** here actually)

**Comment by Acly** [comment]
Thank you for setting this in motion! I've been a bit sceptical about Discord because I find it quite distracting at times, but it's a great way to reach more people who aren't on GitHub. I'm happy to join, just don't expect me to be super responsive there. Putting up a link on website & readme is also easy enough.

**Reply by minsky91** [reply]
That would be most appreciated, if you joined the group. Being small and dedicated to a particular SD art tool, I don’t think it will ever be a source of distraction. But sure, no expectations from your part - just some occasional thumbs up, maybe. The idea is that we will be helping each other with whatever experience we already have (like I, for instance, already did a few times on the SD reddit sub), and if that doesn’t suffice, the person with a plugin-related issue will bring it to this github group, as before. I also imagine, a substantial part of discussions will be actually dedicated to Krita-related issues, a tool which is far from perfect (as I found).

**Comment by sraney** [comment]
I would love to see a Discord support and sharing group, there is so much potential for artists with this.

**Reply by minsky91** [reply]
I am happy to see a reaction like this. Any chance of you willing to be a co-moderator of the group?

**Comment by Amsmolfish** [comment]
Hell yeah. Has there been any progress with this?

**Reply by minsky91** [reply]
The majority here voted for Discord as the platform for the future group. I am not very experienced with it, I am mostly active on the SD subreddit. But no matter which social media, I don’t think it would be wise to run a group without a co-moderator(s). So far, however, no one has contacted me on Discord to offer their support as a co-mod, hence the lack of progress since July. Otherwise, I am still willing to create and lead such a user group.

Perhaps I should repeat the call for co-moderators, by now there might be more people visiting this page.

**Reply by Acly** [reply]
I can create a channel (server) later or tomorrow. Seems like I'm running Discord most of the time these days anyway.

**Reply by minsky91** [reply]
Well, that certainly removes the main obstacle, thank you! Still, a newly created channel must have some text about its goals / subject matter, as well as rules of conduct. I can write that tomorrow by the end of the day.

Will you send me a message over Discord when you are ready with the server? My nick there is minsky91,

**Comment by Acly** [comment]
Join the server here: https://discord.gg/pWyzHfHHhU
